scripts/ubiquiti-discovery.nse

```diff
@@ -0,0 +1,254 @@
+local nmap = require "nmap"
+local shortport = require "shortport"
+local table = require "table"
+local stdnse = require "stdnse"
+local string = require "string"
+local ipOps = require "ipOps"
+
+description = [[
+Extracts information from Ubiquiti networking devices.
+
+This script leverages Ubiquiti's Discovery Service which is enabled by default
+on many products.
+]]
+
+author = {"Tom Sellers"}
+
+license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
+
+categories = {"default", "discovery", "version", "safe"}
+
+---
+-- @usage
+-- nmap -sU -p 10001 --script ubiquiti-discovery.nse <target>
+--
+---
+-- @output
+-- PORT      STATE SERVICE            VERSION
+-- 10001/udp open  ubiquiti-discovery Ubiquiti Discovery Service (ER-X v1.10.7)
+--
+-- | ubiquiti-discovery:
+-- |   uptime_seconds: 84592
+-- |   uptime: 0 days 23:29:52
+-- |   hostname: ubnt-router
+-- |   product: ER-X
+-- |   firmware: EdgeRouter.ER-e50.v1.10.7.5127989.181001.1227
+-- |   version: v1.10.7
+-- |   mac_ip:
+-- |     80:2a:a8:df:a1:63: 192.168.0.1
+-- |     80:2a:a8:df:a1:5e: 55.55.55.55
+-- |   mac_addresses:
+-- |     80:2a:a8:df:a1:63
+-- |_    80:2a:a8:df:a1:5e
+--
+--@xmloutput
+-- <elem key="uptime_seconds">84592</elem>
+-- <elem key="uptime">0 days 23:33:00</elem>
```

Comment:
Should this logically match `uptime_seconds`, just like in the human output?

Reply:
`uptime_seconds` and `uptime`  are both present in human readable and XML output unless I'm overlooking something.

Reply:
What I have meant is that `0 days 23:29:52` is not matching `84592`. In the human output it does.

Reply:
Yes, thanks, good catch.